app/src/main/java/com/orange/ods/app/ui/modules/emptystate/EmptyStateViewModel.kt

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
This is the old copyright header.

Same comment for `EmptyStateSetupScreen.kt`, `EmptyStateDemoScreen.kt` and `OdsTypography.kt`.